src/main/groovy/edu/oregonstate/mist/videogamesampleapi/core/Platform.groovy

```diff
@@ -0,0 +1,18 @@
+package edu.oregonstate.mist.videogamesampleapi.core
+
+/*
+ * Created by George on 12/22/15.
+ */
+
+/*
```

Comment:
The beginning tag of a block comment in java doc has an extra "*". In other words, line 7 should be: "/**". For more information see: https://en.wikipedia.org/wiki/Javadoc